packages/core/src/internal/utils/guards.ts

```diff
@@ -155,23 +149,6 @@ export function isLibrary(
   return future.type === "library";
 }
 
-export function isDeploymentExecutionState(
```

Comment:
These were wrongly outside of our `new-api` folder, and have been moved to `new-api/internal/type-guards.ts`